src/components/organisms/notifier/NotificationEventCreate.tsx

```diff
@@ -0,0 +1,211 @@
+          <GridItem xs>
+            <Typography gutterBottom variant="h6" color="primary">
+              Which event would you like to listen to?
+            </Typography>
+            <Typography gutterBottom color="secondary" variant="body2">
+              Select the event to be listened
+            </Typography>
+            <Select
+              name="eventType"
+              onChange={handleEventTypeChange}
+              value={eventData?.eventType ? eventData.eventType : (SUPPORTED_EVENTS.length ? SUPPORTED_EVENTS[0] : '')}
+              variant="outlined"
+              className={classes.select}
+              id="event-type-select"
+            >
+              {
+        SUPPORTED_EVENTS.map((eventType) => <MenuItem key={eventType} value={eventType}>{notifierEventTypeLabels[eventType]}</MenuItem>)
+      }
+            </Select>
+          </GridItem>
+        </GridRow>
+        <GridRow spacing={4} className={classes.gridRow}>
+          <GridItem>
+            <Typography gutterBottom color="secondary" variant="body2">
+              Smart Contract
+            </Typography>
+          </GridItem>
+          <GridItem>
+            <TextField
+              name="contract"
+              inputRef={register({ required: true })}
+              onChange={handleContractChange}
+              variant="outlined"
+              InputProps={{
+                style: { width: 400, height: 40 },
+              }}
+            />
+            {
+                errors.contract && (
+                <Typography color="error" variant="caption">
+                  Invalid Contract Address
+                </Typography>
+                )
+              }
+          </GridItem>
+        </GridRow>
+        <GridRow spacing={5} className={classes.gridRow}>
+          <GridItem>
+            <Typography gutterBottom color="secondary" variant="body2">
+              Event
+            </Typography>
+          </GridItem>
+          <GridItem>
+            <Box pl={7}>
+              <Select
+                name="eventName"
+                onChange={handleEventChange}
+                value={eventData?.event?.name || (events.length ? events[0].name : '')}
+                className={classes.select}
+                id="event"
+                variant="outlined"
+              >
+                {
+                  events.map((event) => <MenuItem key={event.name} value={event.name}>{event.name}</MenuItem>)
+                }
+              </Select>
+            </Box>
+          </GridItem>
+        </GridRow>
+      </Grid>
+      <Grid>
+        <Box mt={4}>
+          <NotificationChannelsList channels={channels} onEventChannelsUpdate={handleAddChannels} />
+        </Box>
+        <GridRow justify="center" spacing={2}>
+          {
+                eventData?.contract && !eventData?.addedChannels?.length && (
+                <Typography color="error" variant="caption">
+                  Atleast one notification channel must be added
```

Comment:
```suggestion
                  At least one notification channel must be added
```
Maybe we could be friendlier, e.g. "Please add your first channel before continuing."? Or is this in the designs? (sorry didn't look)

Reply:
we have removed this error as the submit button won't be enabled until channels are added